File: TransCoder_tokenized_test_set_functions/java/STERN_BROCOT_SEQUENCE.java
static void SternSequenceFunc ( Vector < Integer > BrocotSequence, int n ) {
  for ( int i = 1;
  BrocotSequence . size ( ) < n;
  i ++ ) {
    int considered_element = BrocotSequence . get ( i );
    int precedent = BrocotSequence . get ( i - 1 );
    BrocotSequence . add ( considered_element + precedent );
    BrocotSequence . add ( considered_element );
  }
  for ( int i = 0;
  i < 15;
  ++ i ) System . out . print ( BrocotSequence . get ( i ) + " " );
}

File: TransCoder_tokenized_test_set_functions/python/STERN_BROCOT_SEQUENCE.py
def SternSequenceFunc ( BrocotSequence, n ) :
    for i in range ( 1, n ) :
        considered_element = BrocotSequence [ i ]
        precedent = BrocotSequence [ i - 1 ]
        BrocotSequence . append ( considered_element + precedent )
        BrocotSequence . append ( considered_element )
    for i in range ( 0, 15 ) :
        print ( BrocotSequence [ i ], end = " " )
    
